tests/e2e/flowtest/flowtest_helpers.py

```diff
@@ -0,0 +1,223 @@
+        osd_pods_before = pod_helpers.get_osd_pods()
+
+        return osd_pods_before, restart_count_before
+
+    def add_capacity_exit_criteria(self, restart_count_before, osd_pods_before):
+        """
+        Exit criteria function for Add capacity operation
+        Args:
+            restart_count_before (dict): Restart counts of pods
+            osd_pods_before (list): List of OSD pods before
+
+        """
+        logger.info("Add capacity exit: Verifying cluster health")
+        self.sanity_helpers.health_check(cluster_check=False, tries=80)
+        logger.info("Add capacity exit: Verifying StorageCluster pods are in running/completed state")
+        wait_for_storage_pods(timeout=300), 'Some pods were not in expected state'
+
+        logger.info("Add capacity exit: Getting restart count of pods after adding capacity")
+        restart_count_after = pod_helpers.get_pod_restarts_count(
+            defaults.ROOK_CLUSTER_NAMESPACE)
+        logging.info(f"sum(restart_count_before.values()) = {sum(restart_count_before.values())}")
+        logging.info(f"sum(restart_count_after.values()) = {sum(restart_count_after.values())}")
+        assert sum(restart_count_before.values()) == sum(restart_count_after.values(
+        )), "Exit criteria verification FAILED: One or more pods got restarted"
+
+        osd_pods_after = pod_helpers.get_osd_pods()
+        number_of_osds_added = len(osd_pods_after) - len(osd_pods_before)
+        logger.info(f"Number of osds added = {number_of_osds_added}, "
+                    f"before = {len(osd_pods_before)}, after = {len(osd_pods_after)}")
+        assert number_of_osds_added == 3, "Exit criteria verification FAILED: osd count mismatch"
+
+        logging.info("Add capacity: Exit criteria verification: Success")
+
+
+@ignore_leftovers
+def create_pvc_delete(multi_pvc_factory, project=None):
+    """
+    Creates and deletes all types of PVCs
+    Returns:
+        bool : True if function runs successfully
+
+    """
+    # Create rbd pvcs
+    pvc_objs_rbd = create_pvcs(
+        multi_pvc_factory=multi_pvc_factory, interface='CephBlockPool',
+        project=project, status="", storageclass=None
+    )
+
+    # Create cephfs pvcs
+    pvc_objs_cephfs = create_pvcs(
+        multi_pvc_factory=multi_pvc_factory, interface='CephFileSystem',
+        project=project, status="", storageclass=None
+    )
+
+    all_pvc_to_delete = pvc_objs_rbd + pvc_objs_cephfs
+
+    # Check pvc status
+    for pvc_obj in all_pvc_to_delete:
+        helpers.wait_for_resource_state(
+            resource=pvc_obj, state=constants.STATUS_BOUND, timeout=300
+        )
+
+    # Start deleting PVC
+    delete_pvcs(all_pvc_to_delete)
+
+    # Check PVCs are deleted
+    for pvc_obj in all_pvc_to_delete:
+        pvc_obj.ocp.wait_for_delete(resource_name=pvc_obj.name)
+
+    logger.info("All PVCs are deleted as expected")
+    return True
+
+
+def obc_put_obj_create_delete(mcg_obj, bucket_factory):
```

Comment:
I think this function should be in sanity_helpers

Reply:
There is one in mcg helpers but it uses awscli pod to write all data, if node running awscli pod is interrupted IOs will stop. This function will just use boto3.

Reply:
I meant that we should move this function to be a sanity_helpers method

Reply:
moved